# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/ladybird/ladybird-exec.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- serenity/Ladybird/CMakeLists.txt.vanilla	2024-04-26 21:47:28.950000000 +0100
+++ serenity/Ladybird/CMakeLists.txt	2024-04-26 21:51:47.465000000 +0100
@@ -247,7 +247,7 @@
 
 function(create_ladybird_bundle target_name)
     set_target_properties(${target_name} PROPERTIES
-        OUTPUT_NAME "Ladybird"
+        OUTPUT_NAME "ladybird"
         MACOSX_BUNDLE_GUI_IDENTIFIER org.SerenityOS.Ladybird
         MACOSX_BUNDLE_BUNDLE_VERSION ${PROJECT_VERSION}
         MACOSX_BUNDLE_SHORT_VERSION_STRING ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}
